docs/changelogs/vNEXT.rst

```diff
@@ -19,14 +19,25 @@ Breaking changes
 .. warning:: This release contains breaking changes. Be sure
    to follow migration steps before upgrading.
 
-
-Breaking change 1
+Breaking changes
 -----------------
 
-1 Behavior of `a(x) if x` has changed
+Behavior of `a(x) if x` has changed
+===================================
+
    * Now equivalant to `a(x) if x == true`
    * Now works if x is unbound
 
+Polar predicates are no longer first class objects
+==================================================
+
+```
```

Comment:
We should add a little more context since this is a breaking change:

Previously it was possible to bind predicates to variables, e.g. `p = f(x)`. This supported an undocumented/not recommended feature for unifying predicates, e.g. `f(x) = f(1)` would unify `x = 1`.

This is no longer legal in this release. E.g. (your example).

A similar thing can be achieved by using lists. Instead of `f(x) = f(1)`, you can write `["f", x] = ["f", 1]` if necessary.